OrbitCore/SamplingProfiler.h

```diff
@@ -92,9 +94,11 @@ class SamplingProfiler {
     return &m_Callstacks;
   }
 
-  const std::unordered_map<CallstackID, std::shared_ptr<CallStack>>&
-  GetUniqueCallstacks() const {
-    return m_UniqueCallstacks;
+  void ForEachUniqueCallstack(std::function<void(const CallStack&)> action) {
```

Comment:
Same line as before, sorry... Can `action` be passed as const-ref?

Reply:
done.